coe/infrastructure-optimization.html

```diff
@@ -1,69 +1,16 @@
 ---
-layout: default
-coe: Infrastructure Optimization
+layout: coe
+title: Infrastructure Optimization
+tagline: Assists in optimizing data centers and reducing infrastructure costs.
+intro: "The Infrastructure Optimization CoE supports agencies in: consolidating federated data centers into a single enterprise facility, modernizing the application landscape, driving out IT infrastructure cost inefficiencies, improving security, and positioning the agency for future infrastructure improvements."
+service-offerings:
+  - Architecture assessments
+  - Optimization planning
+  - Network consolidation
+  - Data center consolidation
+  - New business models e.g. co-location
+sidebar-graphic-class: coe-io
+pdfs:
+  - title: Infrastructure Optimization Service Catalog
+    link: /docs/Infra_Opt_One_Page_Service_Catalog.pdf
 ---
-
-<section class="usa-grid usa-section">
-    <h1>Infrastructure Optimization</h1>
-    <div class="usa-width-two-thirds">
-        <p class="intro">Assists in optimizing data centers and reducing infrastructure costs.</p>
-
-        <div class="page-nav">
-            <a href="#accomplishments">Accomplishments</a> | <a href="#coe-updates">Latest Updates</a>
-        </div>
-        
-     <p>The Infrastructure Optimization CoE supports agencies in: consolidating federated data centers into a single enterprise facility, modernizing the application landscape, driving out IT infrastructure cost inefficiencies, improving security, and positioning the agency for future infrastructure improvements.</p> 
-
-
-        <h2 id="service-offerings">Service Offerings</h2>
-
-        <ul>
-            <li>Architecture assessments</li>
-            <li>Optimization planning</li>
-            <li>Network consolidation</li>
-            <li>Data center consolidation</li>
-            <li>New business models e.g. co-location</li>
-        </ul>
-
-        <h2 id="service-catalog">Service Catalog</h2>
```

Comment:
I removed this section from this page because the content was a duplicate of the data analytics page and seemed like it shouldn't be here -- easy to put back if that's not the case! (Also this is a great example of what the new structure would help avoid.)